src/script/script.h

```diff
@@ -14,6 +14,7 @@
 #include <string.h>
 #include <string>
 #include <vector>
+#include "crypto/common.h"
```

Comment:
So we are now also don't try to differentiate the blocks by own headers, standard headers and library headers?